deploy/crds/csi.ibm.com_ibmblockcsis_crd.yaml

```diff
@@ -1384,7 +1385,7 @@ spec:
             required:
             - controller
             - node
-            type: object
+          type: object
```

Comment:
please align your changes to all crd duplications

Reply:
Done